src/connect.rs

```diff
@@ -161,23 +161,29 @@ impl Connector {
             ProxyScheme::Http { .. } => {
                 unreachable!("connect_socks is only called for socks proxies");
             }
+            ProxyScheme::Https { .. } => {
```

Comment:
This can be `ProxyScheme::Http { .. } | ProxyScheme::Https { .. } => {`.

Reply:
done